src/main/java/study/lotto/domain/LottoNumbers.java

```diff
@@ -0,0 +1,67 @@
+package study.lotto.domain;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class LottoNumbers {
+    static final int LOTTO_NUMBER_SIZE = 6;
+    static final String TO_STRING_DELIMITER = ", ";
+
+    private final List<LottoNumber> value;
+
+    public LottoNumbers(List<Integer> numbers) {
+        checkLottoNumberSize(numbers);
+        checkDuplicates(numbers);
+        value = numbers.stream().map(LottoNumber::new).collect(Collectors.toList());
+    }
+
+    public List<Integer> match(LottoNumbers lottoNumbers) {
+        List<Integer> duplicates = new ArrayList<>(numbers());
+        duplicates.retainAll(lottoNumbers.numbers());
+        return duplicates;
+    }
+
+    public List<Integer> numbers() {
+        return value.stream().map(LottoNumber::toString).map(Integer::valueOf).collect(Collectors.toList());
```

Comment:
```suggestion
        return value.stream()
                .map(LottoNumber::toString)
                .map(Integer::valueOf)
                .collect(Collectors.toList());
```

스트림 사용 시 라인을 나눠서 작성하면 가독성이 상승합니다!

Reply:
네 알겠습니다. 👍  전 보통 한줄에 끝나는 스트림은 그대로 두고 한줄 이상일 경우 stream 다음부터 개행했는데, 한 줄 상관 없이 개행하도록 작성해야겠네요.

Reply:
한 줄 단위로 개행한다면 intelliJ의 힌트도 확인하실 수 있답니다 😄 

![image](https://user-images.githubusercontent.com/40811999/168717206-5f649272-e097-46d7-af8e-299ddecdce17.png)